paasta_tools/kubernetes_tools.py

```diff
@@ -347,6 +348,18 @@ def get_kubernetes_environment(self) -> List[V1EnvVar]:
         ]
         return kubernetes_env
 
+    def get_resource_requirements(self) -> V1ResourceRequirements:
+        return V1ResourceRequirements(
+            limits={
+                'cpu': self.get_cpus() * self.get_cpu_burst_pct() / 100,
+                'memory': self.get_mem(),
```

Comment:
From [the doc](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory) it seems that Kubernetes expects `memory` as a `str` with either a suffixed numeric value or a number of bytes, while [get_mem()](https://github.com/Yelp/paasta/blob/master/paasta_tools/utils.py#L315) returns `float` with Mb.  If that's true then there should be some conversion.
The same for `memory` below.

While here, could you please also double check that `cpu` value is OK here and below.

Reply:
Good catch 👍 I don't think many of our services will run with 1024th of the memory they ask for! Fixup inbound.

I'm pretty sure the CPU stuff is okay, I at least read that bit of the docs pretty closely.